# <mayhem_program::state::BondingCurve as anchor_lang::AccountDeserialize>::try_deserialize  (yours 1000 B at .text+0x25f60; original 1000 B at .text+0x6dd88)
# distance 2 instructions; evidence: called-by anchor_lang::accounts::account::Account<T>::try_from#2752; same-callees 8/8; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -48,8 +48,8 @@
 stxdw [r10-224], r1
 stw [r10-80], 3002
 stb [r10-152], 2
-stw [r10-208], 19
-stdw [r10-216], 34
+stw [r10-208], 24
+stdw [r10-216], 36
 stdw [r10-232], 0
 mov64 r1, r10
 add64 r1, -232
